### ISSUE TITLE
Investigate WCAG 2.1 AA accessibility requirements

### ISSUE THREAD

**Issue by rjkati** [samvera/hyrax#6918]
### Descriptive summary

During the Hyrax Accessibility Community Sprint, look into additional requirements for WCAG 2.1 AA compliance and figure out a plan for further assessment of Hyrax (if needed)

**Comment by rjkati** [comment]
I confirmed that the [DOJ ADA rules require WCAG 2.1](https://www.ada.gov/resources/2024-03-08-web-rule/) 

SiteImprove says that [their accessibility products are based on the Accessibility Conformance Testing (ACT) rules](https://help.siteimprove.com/support/solutions/articles/80000448514-a-guide-to-the-siteimprove-accessibility-checks), which are [compliant with WCAG 2](https://www.w3.org/WAI/standards-guidelines/act/rules/). SiteImprove's statement was last modified on January 30, 2024, so these statements would have been in effect when we did the Hyrax 5.x accessibility audit.

SiteImprove's [automated accessibility checker includes WCAG 2.2](https://help.siteimprove.com/support/solutions/articles/80001136439-siteimprove-now-includes-wcag-2-2-testing), but I cannot locate an equivalent statement for the browser extension. I've submitted a support ticket to SiteImprove to ask and will update this ticket when I hear back.

**Comment by rjkati** [comment]
I did not hear back from SiteImprove before the end of the community accessibility sprint. To cut down on backlog clutter, I will close this ticket and reopen if I do ever hear back.
